src/ashley/editor/decorators.py

```diff
@@ -35,3 +35,27 @@ def emoji(props):
     (https://www.draft-js-plugins.com/plugin/emoji)
     """
     return DOM.create_element("span", {"class": "emoji"}, props["children"])
+
+
+def mention(props):
+    """
+    Decorator for the `mention` entity in Draft.js ContentState.
+
+    This entity is added by the `draft-js-mention-plugin` plugin.
+    (https://www.draft-js-plugins.com/plugin/mention)
+    """
+
+    user = props.get("mention").get("user")
+    name = props.get("mention").get("name")
+
+    if name and user:
+        return DOM.create_element(
+            "a",
+            {
+                "class": "mention",
+                "href": f"/forum/member/profile/{user}/",
```

Comment:
I would find it more explicit to call it `user_id`:
```suggestion
                "href": f"/forum/member/profile/{user_id}/",
```

Reply:
noted ! thanks